Yosemite/Yosemite/Stores/ProductStore.swift

```diff
@@ -17,7 +17,7 @@ public class ProductStore: Store {
         self.init(dispatcher: dispatcher, storageManager: storageManager, network: network, remote: remote)
     }
 
-    init(dispatcher: Dispatcher, storageManager: StorageManagerType, network: Network, remote: ProductsRemoteProtocol) {
+    public init(dispatcher: Dispatcher, storageManager: StorageManagerType, network: Network, remote: ProductsRemoteProtocol) {
```

Comment:
These constructors could be combined in `...Store` classes by making the remote optional and providing a default in the method body, but that might be an over-optimization (at least until the other `...Store` classes have the second constructor?)